### PR TITLE
initial checkin

### PR DESCRIPTION
**Pull request by atracy**
This is an initial checkin of the spiderman ssl documentation

### CODE REVIEW

**Comment by andresriancho** [comment]
@atracy any plans on finishing this PR? Should I complete the work? Would hate to see this work wasted

File: doc/sphinx/complex-web-apps.rst [hunk 1]
```diff
@@ -33,3 +33,62 @@ This is a sample ``spider_man`` plugin run:
...
+    * Tools / Options
+    * Advanced / Certificates
+    * View Certificates
+    * Authorities
+    * Import ...
+    * Browse to the <insert directory here> directory, and click on the file w3af-key.pem , press Open
+    * Select "Trust this CA to identify web sites", and press OK
+    * Close dialogs by pressing OK as necessary
+
+Installing the certificate in Chrome or Internet Explorer
```
Comment (andresriancho) [inline doc/sphinx/complex-web-apps.rst]: w3af is not supported in windows, so just remove internet explorer

File: doc/sphinx/complex-web-apps.rst [hunk 2]
```diff
@@ -33,3 +33,62 @@ This is a sample ``spider_man`` plugin run:
...
+-------------------------------------
+
+Choose the following options:
+
+    * Tools / Options
+    * Advanced / Certificates
+    * View Certificates
+    * Authorities
+    * Import ...
+    * Browse to the <insert directory here> directory, and click on the file w3af-key.pem , press Open
```
Comment (andresriancho) [inline doc/sphinx/complex-web-apps.rst]: what's the directory?

File: doc/sphinx/complex-web-apps.rst [hunk 3]
```diff
@@ -33,3 +33,62 @@ This is a sample ``spider_man`` plugin run:
...
+    * Select "Trust this CA to identify web sites", and press OK
+    * Close dialogs by pressing OK as necessary
+
+Installing the certificate in Chrome or Internet Explorer
+---------------------------------------------------------
+Both Chrome and Internet Explorer use the same trust store for certificates.
+
+    Browse to the <insert directory here> directory, and click on the file w3af-key.pem, and open it
```
Comment (andresriancho) [inline doc/sphinx/complex-web-apps.rst]: What's the directory?

File: doc/sphinx/complex-web-apps.rst [hunk 4]
```diff
@@ -33,3 +33,62 @@ This is a sample ``spider_man`` plugin run:
...
+    Go back to the "General" tab, and click on "Install Certificate ..." and follow the Wizard prompts
+
+Installing the certificate in Opera
+-----------------------------------
+    Tools / Preferences / Advanced / Security
+    Manage Certificates...
+    Select "Intermediate" tab, click "Import..."
+    Browse to the <insert directory here> directory, and click on the file w3af-key.pem, and open it
```
Comment (andresriancho) [inline doc/sphinx/complex-web-apps.rst]: What's the directory?

**Comment by andresriancho** [comment]
@atracy would you like to complete this PR? If you don't send the required fixes I'll close it next week. Thanks!

**Comment by atracy** [comment]
Hi Andres!!  I apologize for not getting back to you earlier!  I've been retasked and swamped with my other work... I don't think I'm going to get to this anytime soon... go ahead and close this issue for now!  Thanks so much!

**Pull request closed without merging**